src/test/java/com/github/sdonkov/PrefixTreeTest.java

```diff
@@ -40,20 +40,21 @@ void testContainsNotAddedWords() throws IOException, URISyntaxException {
     @Test
     void testContainsNull() {
         PrefixTree underTest = new PrefixTree();
-        assertThrows(NullPointerException.class, () -> underTest.contains(null));
+        assertFalse(underTest.contains(null));
     }
 
     @Test
     void testAddNull() {
         PrefixTree underTest = new PrefixTree();
-        assertThrows(NullPointerException.class, () -> underTest.add(null));
+        assertFalse(underTest.add(null));
     }
 
     @Test
     void testAddEmptyString() {
         PrefixTree underTest = new PrefixTree();
-        //todo assert false
-        assertTrue(underTest.add(""));
+        assertAll(  () -> assertFalse(underTest.contains("")),
```

Comment:
I can't see what `assertAll` adds here. My understanding is that it makes sense to use it for checking multiple things regardless of the outcome of each individual check. This seems useful when checking, for example, the contents of a data structure, i.e. it's useful when checking independent statements (for which order is not important). In this case `add` changes the state of the object and affects the next check so it seems like `assertAll` does not fit.